test/cmd/builds.sh

```diff
@@ -112,8 +112,10 @@ echo "patchAnonFields: ok"
 os::test::junit::declare_suite_end
 
 os::test::junit::declare_suite_start "cmd/builds/config"
-os::cmd::expect_success_and_text 'oc describe buildConfigs ruby-sample-build' "Webhook GitHub.+${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/github"
-os::cmd::expect_success_and_text 'oc describe buildConfigs ruby-sample-build' "Webhook Generic.+${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/generic"
+os::cmd::expect_success_and_text 'oc describe buildConfigs ruby-sample-build' "Webhook GitHub:
+.+${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/github"
+os::cmd::expect_success_and_text 'oc describe buildConfigs ruby-sample-build' "Webhook Generic:
+.+${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/generic"
```

Comment:
yes i wanted your input on this one.
for this i tried a normal regex and none worked for me.
the old output was 
Webhook GitHub: http://url

now, because there are multiple the output is

Webhook GitHub:
    http://url1
    http://url2
so the check here no longer matches, i tried all variations for the end of the line char \n \\S \$ and none of them work so i just pressed enter and now it evaluates as a string with a end-of-line char.